engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/ValidatorsEvaluatorChain.scala

```diff
@@ -0,0 +1,20 @@
+package pl.touk.nussknacker.engine.definition.validator
+
+import java.lang.reflect.Parameter
+
+import pl.touk.nussknacker.engine.api.definition.ParameterValidator
+
+object ValidatorsEvaluatorChain {
```

Comment:
I don't think that we need chain for that. Chain is need when you need to find final value based on some chain of logic which can be broken on some step. Here you have a list of extractors, and want to ask them to find all possible validators and merge all results (using ++)

Reply:
Done